app/config/templates/site.php

```diff
@@ -1469,4 +1471,135 @@ function getFramework(string $frameworkEnum, array $overrides)
             ],
         ]
     ],
+    [
+        'key' => 'crm-dashboard-react-admin',
+        'name' => 'CRM dashboard with React Admin',
+        'tagline' => 'A React-based admin dashboard template with CRM features.',
+        'score' => 4, // 0 to 10 based on looks of screenshot (avoid 1,2,3,8,9,10 if possible)
+        'useCases' => [UseCases::DASHBOARD],
+        'screenshotDark' => $url . '/images/sites/templates/crm-dashboard-react-admin-dark.png',
+        'screenshotLight' => $url . '/images/sites/templates/crm-dashboard-react-admin-light.png',
+        'frameworks' => [
+            getFramework('REACT', [
+                'providerRootDirectory' => './react/react-admin',
+                'installCommand' => 'pnpm install && pnpm db-seed',
```

Comment:
Probably better to put DB seed command into `buildCommand` instead of `installCommand`. Neither is best terminology, but install is worse IMO

Reply:
fair enough